packages/twenty-server/src/modules/calendar/calendar-event-import-manager/drivers/utils/sanitize.ts

```diff
@@ -0,0 +1,24 @@
+export const sanitizeObject = <T extends Record<string, any>>(
+  object: T,
+  propertiesToSanitize: (keyof T)[],
+): T => {
+  const sanitizedObject = { ...object };
+
+  for (const property of propertiesToSanitize) {
+    sanitizedObject[property] = sanitizeString(
+      sanitizedObject[property],
+    ) as any;
+  }
+
+  return sanitizedObject;
+};
+
+export const sanitizeString = (
+  value: string | null | undefined,
+): string | null | undefined => {
+  if (value === null || value === undefined || typeof value !== 'string') {
+    return value;
+  }
+
+  return value.replace(/[^\x20-\x7E]/g, '');
```

Comment:
I think this will raise more issue than the initial problem. For example we have chinese users, will this work? Also probably someone will complain about emoji not showing up etc?

Reply:
I would recommend a test driven approach here, let's add a chinese payload and this one and make sure they work :)

Reply:
I'm surprise that this character is an issue: shoudn't it be handled on postgres or typeorm side?

Reply:
@FelixMalfait yes good catch for emojis or aother alphabets. I will try to narrow down to only the breaking char "\u000x00" and retry the import from our buggy customer.

@charlesBochet IMO, It should be the responsability of TypeOrm. Postgres cannot handle this char so the ORM should be able to detect it and strip it. But obviously it does not.